Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Go base image
+FROM golang:1.17-alpine as builder
+
+RUN go get github.com/GeertJohan/go.rice && go get github.com/GeertJohan/go.rice/rice
+
+WORKDIR /app
+
+COPY . .
+
+RUN go install
+
+RUN rice embed-go && go build
+
+FROM alpine:3.14
+
+COPY --from=builder /app/lnme /lnme
+
+EXPOSE 1323
```

Comment:
the PORT can be configured... should this then also be configurable (based on the LNME_LISTEN env variable) or is it a practice to NOT change the default port when using docker? 
(I am not very experienced with docker)

Reply:
EXPOSE just exposes a port to other containers by default.

At runtime / in a higher level compose file you can expose and/or open-to-host any port you want.

This should be fine to use the default port. (It's semi-convention)

Reply:
thanks! 👍

Reply:
From the docker documentation



> The `EXPOSE` instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.